ci/tests/general/general_disk.py

```diff
@@ -196,6 +199,32 @@ def partition_disk(disk):
         return disk.partitions[0]
 
     @staticmethod
+    def unpartition_disk(disk, wait=True):
+        """
+        Return disk to RAW state
+        :param disk: Disk DAL object
+        :return: None
+        """
+        if len(disk.partitions) == 0:
+            pass
+
+        root_client = SSHClient(disk.storagerouter.ip, username='root')
+        for partition in disk.partitions:
+            root_client.run("umount {0}".format(partition.mountpoint))
```

Comment:
Command throws CalledProcessError when mountpoint is not mounted or when mountpoint is None